### PR TITLE
numpy.ndarray as an input for combine [#493]

### PR DESCRIPTION
**Pull request by ysBach**
Please have a look at the following list and replace the "[ ]" with a "[x]" if
the answer to this question is yes.

- [x] For new contributors: Did you add yourself to the "Authors.rst" file?

For new functionality:

- [x] Did you add an entry to the "Changes.rst" file?
- [x] Did you include a meaningful docstring with Parameters, Returns and Examples?
- [x] Does the commit message include a "Fixes #issue_number" (replace "issue_number").
- [x] Did you include tests for the new functionality?
- [ ] Does this PR add, rename, move or remove any existing functions or parameters?

Please note that the last point is not a requirement. It is meant as a check if
the pull request potentially breaks backwards-compatibility.

-----------------------------------------

### CODE REVIEW

**Comment by crawfordsm** [comment]
Thanks so much for the contribution @ysBach !   Looks pretty good to me, but one thing that will need to be added is a test for passing a numpy array to combiner.   This should be added to tests/test_combine.py and be similar to the function `test_combine_average_fitsimages` which is in there if you'd like to see an example.

**Comment by ysBach** [comment]
@crawfordsm Thanks for kind comments :D 
As I am not a developer and not yet used to GitHub or contribution, my PR might not be "usual", so I apologize for that if it is.
After making test, I found one typo from the original commit, so corrected that too. Thanks!

File: CHANGES.rst [hunk 1]
```diff
@@ -24,6 +24,9 @@ New Features
   through to the ``WCS`` method of the same name to allow more flexible
   handing of headers with SIP distortion. [#501]
 
+- ``combine`` now accepts ``numpy.ndarray`` as the input ``img_list``. [#493]
```
Comment (mwcraig) [inline CHANGES.rst]: Very minor request: Could you please also add the pull request number here? So `[#493]` → `[#493, #503]`

File: ccdproc/tests/test_combiner.py [hunk 2]
```diff
@@ -284,6 +284,25 @@ def test_combine_average_fitsimages():
...
+    c = Combiner(ccd_list)
+    ccd_by_combiner = c.average_combine()
+
+    fitsfilename_list = np.array([fitsfile]*3)
+    avgccd = combine(fitsfilename_list, output_file=None, method='average', unit=u.adu)
+    # averaging same fits images should give back same fits image
+    np.testing.assert_array_almost_equal(avgccd.data, ccd_by_combiner.data)
+
+
+
```
Comment (mwcraig) [inline ccdproc/tests/test_combiner.py]: Could you please delete one (only one) of the blank lines here? That will fix the last failing test...

**Comment by ysBach** [comment]
@mwcraig 
Thank you so much for the comments :D 
I have changed the parts as you suggested, and it seems like all check have passed with no error.
I'd be glad to know if there is something more to do regarding this issue from my side :)

**Comment by crawfordsm** [comment]
Thanks @ysBach !

**Pull request merged**